XamlControlsGallery/ControlPages/IconElementPage.xaml

```diff
@@ -0,0 +1,89 @@
+<Page
+    x:Class="AppUIBasics.ControlPages.IconElementPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:animatedvisuals="using:Microsoft.UI.Xaml.Controls.AnimatedVisuals"
+    xmlns:local="using:AppUIBasics"
+    xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
+    mc:Ignorable="d"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+    <StackPanel>
+        <local:ControlExample Name="Example1" HeaderText="A BitmapIcon with a multicolor bitmap image">
+            <local:ControlExample.Example>
+                <StackPanel>
+                <TextBlock TextWrapping="WrapWholeWords">
```

Comment:
The text is right next to the sample and could use a margin. Consider looking at how this is done here: https://github.com/microsoft/Xaml-Controls-Gallery/blob/ec9bedf4624978dd6878c0c97b4b1a03c859c16c/XamlControlsGallery/ControlPages/XamlCompInteropPage.xaml#L14

There is a 12 px margin on bottom.